libs/cardano-ledger-test/cardano-ledger-test.cabal

```
@@ -78,6 +93,7 @@ library
         cardano-protocol-tpraos,
         cardano-slotting,
         containers,
+        hspec-expectations,
```

There is never really a need to import `hspec-expectation` directly, they are all re-exported from `Test.HSpec`
```suggestion
        hspec,
```

---

done